tutorials/inputs/inputevent.rst

```diff
@@ -156,16 +156,20 @@ There are several specialized types of InputEvent, described in the table below:
 Actions
 -------
 
-An InputEvent may or may not represent a predefined action. Actions are
-useful because they abstract the input device when programming the game
-logic. This allows for:
+Actions are a grouping of zero or more InputEvents into a commonly
+understood title (such as "ui_left"). They are not required to represent an
+InputEvent but are useful because they abstract various inputs when
+programming the game logic.
```

Comment:
```suggestion
Actions are a grouping of zero or more InputEvents into a commonly
understood title (such as "ui_left" grouping both joypad-left input and a keyboard's left arrow key). They are not required to represent an
InputEvent but are useful because they abstract various inputs when
programming the game logic.
```